### PR TITLE
Fixes #145 - Add on mouse hover date/time for alerts

### LINKED ISSUE #145: Show absolute date and time on hover in popup window
It would be amazing if we can see time and date on mouse hover in popup window - now we can see only Today, Yesterday, XX hours but we cannot see absolute time - so hover on the date it would show tooltip with absolute date and time

just title="" would be enough

reguest

EDIT: unfortunately mouse cursor is invisible, it is pointing to the first row

### PR DESCRIPTION
**Pull request by Eldenroot**
Plus some small changes in a design.

Fixes #145

### CODE REVIEW

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html [hunk 1]
```diff
@@ -14,8 +14,8 @@
         <br class="clear"/>
     </td>
     <td class="{$altbg} alert__time" align="center">
-        {$alert['received_at']}<hr>
-        <a href="{$mybb->settings['bburl']}/alerts.php?action=markread&amp;id={$alert['id']}&amp;my_post_key={$mybb->post_code}" class="markReadAlertButton" id="markread_alert_{$alert['id']}" title="{$lang->myalerts_page_row_read_title}">{$lang->myalerts_page_row_read}</a> | 
-        <a href="{$mybb->settings['bburl']}/alerts.php?action=delete&amp;id={$alert['id']}&amp;my_post_key={$mybb->post_code}" class="deleteAlertButton" id="delete_alert_{$alert['id']}" title="{$lang->myalerts_page_row_delete_title}">{$lang->myalerts_page_row_delete}</a>
+      <span class="smalltext">{$alert['received_at']}</span><hr>
```
Comment (lairdshaw) [inline inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html:17]: This styling does not seem to be relevant to this PR, but maybe I'm missing something: is there a reason why the change of date format to `relative` requires this style change?

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html [hunk 2]
```diff
@@ -14,8 +14,8 @@
...
+      <a href="{$mybb->settings['bburl']}/alerts.php?action=markread&amp;id={$alert['id']}&amp;my_post_key={$mybb->post_code}" class="markReadAlertButton" id="markread_alert_{$alert['id']}" title="{$lang->myalerts_page_row_read_title}">{$lang->myalerts_page_row_read}</a> | 
+      <a href="{$mybb->settings['bburl']}/alerts.php?action=delete&amp;id={$alert['id']}&amp;my_post_key={$mybb->post_code}" class="deleteAlertButton" id="delete_alert_{$alert['id']}" title="{$lang->myalerts_page_row_delete_title}">{$lang->myalerts_page_row_delete}</a>
```
Comment (lairdshaw) [inline inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html:19]: The indentation changes of the above two lines also do not seem to belong in this PR. Can you please revert them?

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row_popup.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
         </a>
     </td>
     <td class="{$altbg} alert__time">
-        {$alert['received_at']}<br />
+        <span class="smalltext">{$alert['received_at']}</span><br />
```
Comment (lairdshaw) [inline inc/plugins/MybbStuff/MyAlerts/templates/alert_row_popup.html:16]: As above: this styling does not seem to be relevant to this PR. Same question as above.

**Comment by Eldenroot** [comment]
Changed, just core changes... thx!
